droopler.install

```diff
@@ -2256,3 +2256,12 @@ function droopler_update_8138(&$sandbox) {
   \Drupal::service('d_update')->checklist()
     ->saveProgress(['v3.1' => $values]);
 }
+
+/**
+ * Droopler 3.3 Change AdvAgg JS minifier from JSqueeze to JShrink.
+ */
+function droopler_update_8139(&$sandbox) {
```

Comment:
Maybe the update number should be 10001? But then maybe there can be an issue with overlapping number if somebody will update to Droopler 4?